gaia-header.js

```diff
@@ -91,6 +91,11 @@ module.exports = component.register('gaia-header', {
       titles: this.getElementsByTagName('h1')
     };
 
+    // The [ignoreDir] attribute can be used to force the older behavior of
+    // gaia-header, where the whole header is always displayed in LTR mode.
+    this.ignoreDir = this.hasAttribute('ignoreDir');
```

Comment:
nit: use `ignore-dir` instead of `ignoreDir`, to follow the style convention of custom elements and custom attributes.